install.sh

```diff
@@ -485,33 +490,45 @@ configure_linux_token() {
 postinstall_check() {
   detect_mondoo
   if [ $MONDOO_INSTALLED = false ]; then
-    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the Mondoo binary)."
+    red "${MONDOO_PRODUCT_NAME} installation failed (can't find the ${MONDOO_BIN} binary)."
     exit 1
   fi
 
   echo "${MONDOO_PRODUCT_NAME} installation completed."
 }
 
 finalize_setup() {
+
+  # If registration token is provided, register client
   configure_token
 
   # Display final message
-  detect_mondoo_registered
-  if [ $MONDOO_IS_REGISTERED = false ]; then
-    purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
-    echo
-    lightblue_bold "Next you should register ${MONDOO_PRODUCT_NAME} to get access to policies and reports."
-    lightblue_bold "Follow this guide: "
+  purple_bold "\n${MONDOO_PRODUCT_NAME} is ready to go!"
+
+  # Only if installing Mondoo Client, warn user to register. Do not warn open source users.
+  if [ "$MONDOO_PRODUCT_NAME" = "Mondoo Client" ]; then
+    detect_mondoo_registered
+    if [ $MONDOO_IS_REGISTERED = false ]; then
+      echo
+      lightblue_bold "Next you should register ${MONDOO_PRODUCT_NAME} to get access to policies and reports."
+      lightblue_bold "Follow this guide: "
+      echo
+      lightblue_bold "https://mondoo.com/docs/operating_systems/installation/registration/"
+      echo
+    else
+      echo
+      lightblue_bold "Run 'mondoo scan local' to scan this system or learn more in our quick start docs: https://mondoo.com/docs/"
```

Comment:
Suggest the super simple form:

```suggestion
      lightblue_bold "Run 'mondoo scan' to scan this system or learn more in our quick start docs: https://mondoo.com/docs/"
```

Reply:
This doesn't work for me, it just prints help.

Reply:
```
❯ mondoo version
Mondoo 7.3.0 (c00b02ace, 2022-11-08T19:41:00Z)
❯ mondoo scan

This command triggers a new policy scan for an asset. By default, the local
system is scanned with its pre-configured policies:

    $ mondoo scan local

Users can also manually select a local policy to execute and run it without
storing results in the server:

    $ mondoo scan local --policy-bundle policyfile.yaml --incognito
...
```

Reply:
🤦 You're right. We only fixed that in cnspec.